measure/src/macros.rs

```diff
@@ -0,0 +1,112 @@
+/// macro that wraps the Measure::this function to make its usage more ergonomic
+///
+/// Use `measure!()` macro when you have a function that you want to measure.
+/// It has identical functionality to the Measure::this() call.
+/// It will start a new `Measure`, call your function, stop the measure, then return the `Measure`
+/// object along with your function's return value.
+///
+/// # Examples
+/// ```
+/// // With macro
+/// # use solana_measure::measure;
+/// # fn zero() {};
+/// # let (result, measure) = measure!(zero(), "zero params");
```

Comment:
Probably wanna see this line
```suggestion
/// let (result, measure) = measure!(zero(), "zero params");
```